hadoop-ozone/ozone-recon/src/test/java/org/apache/hadoop/ozone/recon/api/TestUtilizationService.java

```diff
@@ -62,66 +64,73 @@ public void setUpResultList() {
             (long) i));
       }
     }
+    return resultList;
   }
 
   @Test
   public void testGetFileCounts() throws IOException {
-    setUpResultList();
+    List<FileCountBySize> resultList = setUpResultList();
 
     utilizationService = mock(UtilizationService.class);
     when(utilizationService.getFileCounts()).thenCallRealMethod();
     when(utilizationService.getDao()).thenReturn(fileCountBySizeDao);
     when(fileCountBySizeDao.findAll()).thenReturn(resultList);
 
-    utilizationService.getFileCounts();
+    Response response = utilizationService.getFileCounts();
+    // get result list from Response entity
+    List<FileCountBySize> responseList =
+        (List<FileCountBySize>) response.getEntity();
+
     verify(utilizationService, times(1)).getFileCounts();
```

Comment:
Why are we verifying the actual method call? Method calls verification is generally used for mocked methods (So that we know the code path went through that).